web/src/features/charts/hooks/useBarElectricityBreakdownChartData.ts

```diff
@@ -56,12 +57,15 @@ export default function useBarBreakdownChartData() {
   );
   const height = isConsumption ? exchangeY + exchangeHeight : exchangeY;
 
+  const emissionSourceToProductionSource = getEmissionData(zoneData);
```

Comment:
I'm not sure on the naming here, is this a function that converts an emission source to a production source? If it's a variable then the word "to" implies it converts and could be rephrased

Reply:
I have named it `sourceProductionSourceMapping` now. I searched a bit on naming conventions for Maps and a lot of suggestions have "to" in it. I think that `sourceToProductionSourceMapping` might be better, what do you think?